include/deal.II/base/mpi_remote_point_evaluation.h

```diff
@@ -113,8 +113,40 @@ namespace Utilities
       /**
        * Data of points positioned in a cell.
        */
-      struct CellData
+      class CellData
```

Comment:
We will extract the changes related to `CellData` into a separate PR.

Reply:
Very good idea!